Automattic/wp-calypso
# DevDocs: New "Gutenberg components" section

First task for exposing the Gutenberg components in Calypso and part of #26180.

It creates a new section in the DevDocs that displays the collection of components available in Gutenberg, with documentation explaining how to use them.

Right now, the collection only features the `Button` component (rest of them will be included in the next PR).

### Testing Instructions
- Navigate to `/devdocs`
- New section "Gutenberg Components" appears in the sidebar under "Live Docs"
- Click on the "Gutenberg Components" section
- It redirects to `/devdocs/gutenberg-components` and a page with a short intro and a collection of component appears
- The short introduction has a link that redirects to documentation with more info
- The gallery only features the `Button` components
- Click on the `Button` link
- It redirects to a page that displays the same example as before, the code used in the example and README.md file included in the `@wordpress/components` package for the `Button` component (https://github.com/WordPress/gutenberg/blob/master/packages/components/src/button/README.md)

### Screenshots
![screen shot 2018-07-25 at 23 16 58](https://user-images.githubusercontent.com/1233880/43228109-fa149eb8-9060-11e8-84db-d873cf963085.png)
![screen shot 2018-07-25 at 23 17 11](https://user-images.githubusercontent.com/1233880/43228110-fa2e4958-9060-11e8-89d5-56a5d7c814bd.png)
![screen shot 2018-07-25 at 23 17 25](https://user-images.githubusercontent.com/1233880/43228111-fa481036-9060-11e8-87e6-720269658ae9.png)
